microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -5,15 +5,11 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 08/18/2023
+ms.date: 12/13/2023
 ---
 # Microsoft Edge Add-ons store developer policies
 
-
-<!-- ====================================================================== -->
-## Introduction and Objective of this document
-
-Thank you for your interest in developing extensions for the Microsoft Edge Add-ons store.  The Microsoft Edge Add-ons store developer policies (Add-ons store developer policies) apply to your extensions, including your submission of extensions through [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd) and the provision of such extensions through the Microsoft Edge Add-ons.
+Thank you<!-- todo: go easy on "thank you", per corp-level style guide --> for your interest in developing extensions for the Microsoft Edge Add-ons store.  The Microsoft Edge Add-ons store developer policies (Add-ons store developer policies) apply to your extensions, including your submission of extensions through [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd) and the provision of such extensions through the Microsoft Edge Add-ons.
```

Comment:
```suggestion
Thanks for your interest in developing extensions for the Microsoft Edge Add-ons store.  The Microsoft Edge Add-ons store developer policies (Add-ons store developer policies) apply to your extensions, including your submission of extensions through [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd) and the provision of such extensions through the Microsoft Edge Add-ons.
```

Reply:
Writer/Editor proposal: strike "Thanks for your interest in developing extensions for the Microsoft Edge Add-ons store." - it's tangential and not substantive for this article; it's low-utility info content.  Then smooth out the remainder of intro paragraph.

Reply:
used wording:
To develop an extension to be published through the Microsoft Edge Add-ons store, follow these Microsoft Edge Add-ons store developer policies (_Add-ons store developer policies_), which apply to submitting extensions through [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd) and providing extensions through the Microsoft Edge Add-ons store.